### PR TITLE
Simplify texts in various dialogs, for translators' benefit

### PR DESCRIPTION
**Pull request by Jojo-Schmitz**
by eliminating html stuff from the to be translated texts.
Move URLs out of translations, which is also good for security reasons, to protect us from bogus and malicous translations.
Some formatting changes and getting some error messages to stderr.
Updating copyright to 2016 in the due course.

### CODE REVIEW

File: mscore/musescore.cpp [hunk 1]
```diff
@@ -2967,6 +2967,12 @@ AboutBoxDialog::AboutBoxDialog()
       setWindowFlags(this->windowFlags() & ~Qt::WindowContextHelpButtonHint);
 
       copyRevisionButton->setIcon(*icons[int(Icons::copy_ICON)]);
+
+      copyrightLabel->setText(QString("<span style=\"font-size:10pt;\">%1</span>")
```
Comment (lasconic) [inline mscore/musescore.cpp]: I would go further and not include any HTML marking in the translation. http://musescore.org will redirect to the right language.
The BR can be hardcoded.
The donation page doesn't see to the language selection automatically, we could ask Thomas to do something about it.

Comment (Jojo-Schmitz) [inline mscore/musescore.cpp]: yes, I was contemplating this, also for security reasons, a bogus translation could redirect to some entirely different place.
There are other places though where HTML is in the translations, like the save onlyin dialogs and some error messages

Comment (Jojo-Schmitz) [inline mscore/musescore.cpp]: hadrcoding the BR would mean to have several separate strings to translate rather than just one, and with context

Comment (lasconic) [inline mscore/musescore.cpp]: I stil think it would be better than having bad HTML coming from translation.

Comment (Jojo-Schmitz) [inline mscore/musescore.cpp]: Different translations may requite different linefeeds

**Comment by Jojo-Schmitz** [comment]
I've taken all URLs out from the translations now, should be safer, worst thing is that links don't work at all, when the %1...%2 get lost in Translation by accident.
Well, or that someone hardcodes a URL into the translations...

@thomasbonte: could you do something about redirecting the donations page to the national one?
Like http://musescore.org/donations -> http://musescore.org/en/donation, http://musescore.org/de/spenden, ...

File: mscore/musescore.cpp (hunk 1 above)
Comment (Jojo-Schmitz) [inline mscore/musescore.cpp]: BTW the redirection to the right language only works when MuseScore's language is set to System (or equivalent)

Comment (Jojo-Schmitz) [inline mscore/musescore.cpp]: I may have found a method to get rid of the <br>, actually 2 methods, a) via css `whit-space:pre-wrap` or for `.replace("\n", "<br/>")`, which would you prefer?

Comment (Jojo-Schmitz) [inline mscore/musescore.cpp]: I'll go for the replace method, it is easier to read and use

**Pull request merged**